singer_sdk/connectors/sql.py

```diff
@@ -871,6 +860,87 @@ def _get_column_type(
 
         return cast(sqlalchemy.types.TypeEngine, column.type)
 
+    @staticmethod
+    def get_column_create_ddl(
```

Comment:
Should we rename this to `get_column_add_add()`? The verb `CREATE` is normally applied to tables, but I think we generally use `ADD` for columns. Wdyt?

I don't have a strong opinion here, so feel free to chose what feels best. 🤷